common/app/com/commercetools/sunrise/framework/reverserouters/myaccount/recoverpassword/SimpleRecoverPasswordReverseRouter.java

```diff
@@ -23,4 +23,8 @@
     String RESET_PASSWORD_PROCESS = "resetPasswordProcessCall";
 
     Call resetPasswordProcessCall(final String languageTag, final String resetToken);
+
+    String RESET_PASSWORD_SUCCESS = "resetPasswordSuccessCall";
+
+    Call resetPasswordSuccessCall(final String languageTag);
```

Comment:
The reset/recovery naming is always quite confusing, but here I would call it `requestRecoveryEmailSuccessCall`. Notice that this is the success scenario from the `requestRecoveryEmailProcessCall` (when you submit your email), not from the `resetPasswordProcessCall` (which would be when you change your password).

Reply:
Yeo i agreed with you 👍 
i will change the name accordingly  to be in harmony with the naming scheme